app/views/admin/badges/index.html.erb

```diff
@@ -2,7 +2,6 @@
   <div class="col">
     <div class="row justify-content-end">
       <%= link_to "Create Badge", new_admin_badge_path, class: "btn btn-primary" %>
-      <%= link_to "Award Badge", admin_badge_achievements_award_badges_path, style: "margin-left: 20px;", class: "btn btn-primary" %>
```

Comment:
Hmm I actually think it's nice to keep this here in `/admin/badges` 😅

Reply:
Okay, I've added this back! 😄